pandas/tests/groupby/aggregate/test_cython.py

```diff
@@ -336,15 +336,9 @@ def test_cython_agg_nullable_int(op_name):
         }
     )
     result = getattr(df.groupby("A")["B"], op_name)()
-    df2 = df.assign(B=df["B"].astype("float64"))
+    df2 = df.assign(B=df["B"])
```

Comment:
I think we want to still use NumPy dtypes here, then cast expected to the expected nullable dtype.

Reply:
Ah okay. I put back this cast and used `convert_dtypes` to cast expected to the nullable dtype